GDJS/Runtime/runtimegame.ts

```diff
@@ -143,6 +146,17 @@ namespace gdjs {
       this._sessionId = null;
       this._playerId = null;
 
+      this._embeddedResourcesMappings = new Map();
+      for (const resource of this._data.resources.resources) {
+        if (resource.metadata) {
+          const metadata = JSON.parse(resource.metadata);
```

Comment:
Probably a good idea to try/catch this (+ log an error) to avoid crashing the game if there is a rogue resource.

Reply:
yep

Reply:
I've added the checks.